seatunnel-connectors-v2/connector-hbase/src/main/java/org/apache/seatunnel/connectors/seatunnel/hbase/format/HBaseDeserializationFormat.java

```diff
@@ -60,16 +60,15 @@ private Object deserializeValue(SeaTunnelDataType<?> typeInfo, byte[] cell) {
             case TINYINT:
             case SMALLINT:
             case INT:
-                return Bytes.toInt(cell);
+                return Integer.valueOf(Bytes.toString(cell));
             case BOOLEAN:
-                return Bytes.toBoolean(cell);
+                return Boolean.valueOf(Bytes.toString(cell));
```

Comment:
**Security Issue**: Converting byte array to Boolean using Bytes.toString(cell) can lead to unexpected results if the byte array contains values other than 'true' or 'false'. <br> **Fix**: Validate the byte array content to ensure it only contains 'true' or 'false' before conversion. <br> **Code Suggestion**: 
 ```
 - return Boolean.valueOf(Bytes.toString(cell));
 + String booleanStr = Bytes.toString(cell);
 + if ("true".equalsIgnoreCase(booleanStr) || "false".equalsIgnoreCase(booleanStr)) {
 +     return Boolean.valueOf(booleanStr);
 + } else {
 +     throw new IllegalArgumentException("Invalid byte array for Boolean conversion");
 + }
 ```